test/adapterManager.spec.js

```diff
@@ -1,402 +1,335 @@
-            sinon.spy(UTIL, 'log');
-            done();
-        });
-
-        afterEach(function(done) {
-            UTIL.isFunction.restore();
-            UTIL.log.restore();
-            bidAdaptor.ID.restore();
-            bidAdaptor = null;
-            adapterID = null;
-            done();
-        });
-
-        it('is a function', function(done) {
-            ADPTMgr.registerAdapter.should.be.a('function');
-            done();
-        });
-
-        it('should call UTIL.log if bidAdaptor is not an object', function(done) {
-            ADPTMgr.registerAdapter(null);
-            UTIL.log.calledWith(CONSTANTS.MESSAGES.M3).should.be.true;
-            UTIL.log.calledWith(null).should.be.true;
-            done();
-        });
-
-        it('should have called UTIL isFunction and bidAdaptor.ID', function(done) {
-            ADPTMgr.registerAdapter(bidAdaptor);
-            UTIL.isFunction.calledOnce.should.be.true;
-            bidAdaptor.ID.called.should.be.true;
-            done();
-        });
-
-        it('should have added bidAdaptor into registeredAdapters if bidAdaptor has fB function with given adapterID', function(done) {
-            ADPTMgr.registerAdapter(bidAdaptor);
-            ADPTMgr.registeredAdapters[adapterID].should.deep.equal(bidAdaptor);
-            done();
-        });
-
-
-        it('should have caled UTIL.log if bidAdaptor doesnt have fB function', function(done) {
-            delete bidAdaptor.fB;
-            ADPTMgr.registerAdapter(bidAdaptor);
-            UTIL.log.calledOnce.should.be.true;
-            ADPTMgr.registeredAdapters[adapterID].should.not.equal(bidAdaptor);
-            UTIL.log.calledWith(bidAdaptor.ID() + CONSTANTS.MESSAGES.M3).should.be.true;
-            done();
-        });
-    });
-    /* end-test-block */
-
-
-    /* start-test-block */
-    describe('#registerAdapters', function() {
-
-
-        beforeEach(function(done) {
-            sinon.spy(ADPTMgr, "registerAdapter");
-            sinon.spy(prebid, 'register');
-            done();
-        });
-
-        afterEach(function(done) {
-            ADPTMgr.registerAdapter.restore();
-            prebid.register.restore();
-            done();
-        });
-
-        it('is a function', function(done) {
-            ADPTMgr.registerAdapters.should.be.a('function');
-            done();
-        });
-
-        it('should have called ADPTMgr.registerAdapter and prebid.register', function(done) {
-            ADPTMgr.registerAdapters();
-            ADPTMgr.registerAdapter.calledOnce.should.be.true;
-            prebid.register.calledOnce.should.be.true;
-            done();
-        });
-    });
-    /* end-test-block */
-});
+/* global describe, it, xit, sinon, expect */
+// var sinon = require("sinon");
+var should = require("chai").should();
+var expect = require("chai").expect;
+
+var ADPTMgr = require('../src_new/adapterManager');
+var CONFIG = require("../src_new/config.js");

```

Comment:
![Codacy](https://codacy.pubmatic.com:443/assets/images/favicon.png) Issue found: ['CONFIG' is assigned a value but never used.](https://codacy.pubmatic.com:443/manual/dattaprasad.mundada/OpenWrap/pullRequest?prid=15196)